### ISSUE TITLE
Default box scale of DSSD

### ISSUE THREAD

**Issue by youngwanLEE** [weiliu89/caffe#624]
Hi, Wei. 

I have implemented your DSSD on VOC and ILSVRC16.

I implemented PM(Prediction Module) + DM(Deconvolution Module) based on SSD-VGG-512 model and achieve mAP 80.87 on VOC07test (trained on 07+12 from scratch). 

However, when implementing the original ResNet101-based SSD,  the accuracy is 77 mAP.

My settings are belows : 

1. 1. added 1.6 default box aspect ratio setting 
2. 2. used conv3_x, conv5_x featureMap referred to [your code.](https://github.com/weiliu89/caffe/blob/ssd/examples/ssd/ssd_pascal_resnet.py#L258).  
3. used same default box  scale in your model(SSD-300, SSD-512 models).
4. used same learning rate policy( init_learning rate : 0.001 and multi-step)

I tried to experiment many trials. But the result is as not good as your results on your paper.

I guesses the default box scale setting. 

Could you advice for me?

Thanks in advance.

**Comment by KeyKy** [comment]
@youngwanLEE I found your WR-Inception+SSD. Does it better than VGG16 in human detection? Thanks.

**Comment by youngwanLEE** [comment]
@KeyKy yes,  of course, various strategies for KITTI are needed.

**Comment by KeyKy** [comment]
@youngwanLEE Sorry to bother you again, Where do you get the ImageNet pretrained model of WR-Inception?

**Comment by youngwanLEE** [comment]
@KeyKy I directly trained pretrained model.

**Comment by roipony** [comment]
@Keyky can you share DSSD code? 
Thanks

**Comment by KeyKy** [comment]
@roipony I only talk about SSD here.

**Comment by roipony** [comment]
@youngwanLEE can you share DSSD code?
Thanks

**Comment by CrazySssst** [comment]
@youngwanLEE  Do you implement ResNet+SSD successfully?

**Comment by isalirezag** [comment]
can you please explain to  me what are the meaning of min_ratio and step and aspect_ratios
